spamgourmet-clone/scripts/install-sg-exim.sh

```diff
@@ -27,7 +27,7 @@ function exim4_setup {
   sed -i "s/dc_other_hostnames=.*$/dc_other_hostnames='$DOMAIN'/" /etc/exim4/update-exim4.conf.conf
   sed -i "s/dc_local_interfaces=.*$/dc_local_interfaces='$IPADDRESS'/" /etc/exim4/update-exim4.conf.conf
   sed -i "s/dc_readhost=.*$/dc_readhost=''/" /etc/exim4/update-exim4.conf.conf
-  sed -i "s/dc_relay_domains=.*$/dc_relay_domains='ob."$DOMAIN"'/" /etc/exim4/update-exim4.conf.conf
+  sed -i "s/dc_relay_domains=.*$/dc_relay_domains='ob.$DOMAIN'/" /etc/exim4/update-exim4.conf.conf
```

Comment:
Regarding this method for creating configuration file, I also have a few configs modified on the fly with `sed`. I recently had a problem where a developer changed the config slightly in a way that made some of the `sed` commands fail. The problem can be avoided by using some kind of templating, e.g. Perl Text::Template. Then the config files with the templating expressions are rename to `.conf.in` and the templates are cleanly replaced to produce the `.conf` without risks.

Thoughts?

Reply:
I've added this to the existing ticket to clean up templating. https://github.com/spamgourmet/spamgourmet/issues/52